### ISSUE TITLE
Fail to import six.moves.urllib on IronPython 2.7.5

### ISSUE THREAD

**Issue by utgwkk** [benjaminp/six#192]
On IronPython 2.7.5:

```
>ipy -X:Frames
IronPython 2.7.5 (2.7.5.0) on .NET 4.0.30319.1 (32-bit)
Type "help", "copyright", "credits" or "license" for more information.
>>> from six.moves import urllib
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Program Files (x86)\IronPython 2.7\lib\site-packages\six.py", line 92, in __get__
  File "C:\Program Files (x86)\IronPython 2.7\lib\site-packages\six.py", line 115, in _resolve
  File "C:\Program Files (x86)\IronPython 2.7\lib\site-packages\six.py", line 82, in _import_module
ImportError: No module named urllib
```

On Python 2.7.11:

```
>python
Python 2.7.11 (v2.7.11:6d1b6a68f775, Dec  5 2015, 20:40:30) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from six.moves import urllib
>>> dir(urllib)
['error', 'parse', 'request', 'response', 'robotparser']
```

Both using six 1.10.0

**Comment by utgwkk** [comment]
Fixed on IronPython 2.7.7

**Comment by benjaminp** [comment]
An IronPython bug then?
